roles/node_exporter/meta/main.yml

```diff
@@ -0,0 +1,28 @@
+galaxy_info:
+  author: Michael Kaye
+  description: Deploys node_exporter from github
+  company: New Vector
```

Comment:
So far I've always put New Vector Ltd there, because that's the actual legal entity. While I don't care which one we do, I think we should keep it consistent.